packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -667,6 +676,22 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         return _getFeeAdjustedCollateral(rawTotalPositionCollateral);
     }
 
+    /**
+     * @notice Accessor method to calculate a transformed price using the provided finanicalProductLibrary specified
+     * during contract deployment. If no library was provided then no modification to the price is done.
+     * @param price input price to be transformed.
```

Comment:
Is it fair to add a `@dev` note that this method should never revert?

Reply:
+1

Reply:
done